lib/mix/tasks/guardian_db.gen.migration.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Mix.Tasks.GuardianDb.Gen.Migration do
+  @shortdoc "Generates GuardianDb's migration"
+
+  @moduledoc """
+  Generates the required GuardianDb's database migration
+  """
+  use Mix.Task
+
+  alias Mix.Generator
+
+  @doc false
+  def run(_args) do
+    if Mix.Project.umbrella?() do
+      Mix.raise "mix guardian_db.gen.migration can only be run inside an application directory"
+    end
+
+    Generator.create_directory("./priv/repo/migrations")
+
+    source_path =
+      :guardian_db
+      |> Application.app_dir()
+      |> Path.join("priv/templates/migration.exs")
+
+    generated_file = EEx.eval_file(source_path, [module_prefix: app_module()])
+    target_file = Path.join("priv/repo/migrations", "#{timestamp()}_guardiandb.exs")
```

Comment:
@yordis are you referring to this filename?

Reply:
The file name of the template yes `priv/templates/migration.exs.eex`

Reply:
You're right, I renamed it.